src/core/unit.c

```diff
@@ -2220,7 +2223,7 @@ static int unit_log_resources(Unit *u) {
         t = strjoina(u->manager->invocation_log_field, u->invocation_id_string);
         iovec[n_iovec + 3] = IOVEC_MAKE_STRING(t);
 
-        log_struct_iovec(log_level, iovec, n_iovec + 4);
+        log_unit_struct_iovec(u, log_level, iovec, n_iovec + 4);
```

Comment:
given this whole function is relatively expensive, it might make sense to add an early-exit here too if the log level would suppress the output anyway